### ISSUE TITLE
Regular/escaped identifiers as infix ?

### ISSUE THREAD

**Issue by fblanqui** [Deducteam/lambdapi#255]
Why not allow regular or escaped identifiers to be declared as infix too? After an infix declaration, one could still access the symbol with no arguments by enclosing it within parentheses like in OCaml. This would allow to write something like:
```
symbol or : ...
set infix left 5 or
```
or better:
```
symbol infix left 5 or : ...
```

**Comment by rlepigre** [comment]
That would be ambiguous since `x or y` could be parsed both as:
- `x` applied to variables `or` and `y`,
- the infix operator `or` applied to `x` and `y`.

**Comment by fblanqui** [comment]
Fixed in #516.
